roadmap.md

```diff
@@ -2,15 +2,35 @@
 
 This page describes planned work for Ignition Robotics. The set of planned
 features and development efforts should provide insight into the overall
-direction of Ignition Robotics. Development of Ignition Robotics is closely
-tied to on-going work with a variety of institutions. If you would like to
-see other features on the roadmap, then please get in touch with us via
+direction of Ignition Robotics. If you would like to
+see other features on the roadmap, then please get in touch with us at
 info@openrobotics.org. 
 
-## Feature Development
+## Quarterly Roadmap
 
-See the [Release Features](/docs/release-features) page for a list of features
-already available in each release of Ignition.
+**Close the Gap** is Ignition's current focus topic. Over the next two
```

Comment:
How about linking to Citadel's [comparison page](https://ignitionrobotics.org/docs/citadel/comparison)? Alternatively, I can see that going with "Feature parity" on line 27 below.

That page is being updated on https://github.com/ignitionrobotics/docs/pull/26. I hope to have is done by the end of next week.

Reply:
Good idea, added in 0c06ae22b68059f67b4caf4abebadd3f881a4464